src/docs/utils/publish_docs.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash -ex
+
+trace_suspend() {
+    case "$-" in *x*)
+        set +x
+        __resume_trace=1
+        ;;
+    esac 2>/dev/null
+}
+
+DOCS_GIT_URL='https://git.launchpad.net/~ubuntu-docker-images/ubuntu-docker-images/+git/templates'
+
+image_name="${1}"
+image_doc_filename="${1}_doc_data"
```

Comment:
The image_doc_filename is because we call the file like this on the generate_oci_documentation script and store it in data. Now I pass it as arguments